### PR TITLE
Feature/mx 1521 add entity type

### PR DESCRIPTION
**Pull request by carolinscholl**
# Ticket Number
MX-1521

# Changes

## Added
The attribute "entityType" was added in all merged and extracted models. It contains the name of the class and is useful for the backend API ($type attribute).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,6 +29,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security
 
+## [0.19.0] - 2024-01-??
+
+### Added
+
+- add `entityType` field in all extracted and merged models
+
+
```
Comment (cutoffthetop) [inline CHANGELOG.md:38]:
```suggestion
```

i think this needs to go under [[Unreleased] -> Added](https://github.com/robert-koch-institut/mex-common/blob/main/CHANGELOG.md?plain=1#L17)

version 19 and the date are only added once we do the actual release

File: pyproject.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "mex-common"
-version = "0.18.1"
+version = "0.19.0"
```
Comment (cutoffthetop) [inline pyproject.toml:3]:
```suggestion
version = "0.19.dev0"
```

if we are doing releases [by the book](https://packaging.python.org/en/latest/specifications/version-specifiers/#developmental-releases) now, it'd have to be something like this. but let's discuss in the office.

**Review by cutoffthetop — DISMISSED** [review]
straight to the point 🎯 

only 2 comments, but those are more in regards to changelog/semver things

File: pyproject.toml (hunk 2 above)
Comment (carolinscholl) [reply pyproject.toml:3]: As discussed, let's stick to the previous release process for this PR and come up with a strategy for future releases including the format of the changelog together on Tuesday?

File: CHANGELOG.md (hunk 1 above)
Comment (carolinscholl) [reply CHANGELOG.md]: I set the date to today cause hopefully I can make the release today 🙏

**Review by cutoffthetop — APPROVED** [review]

**Pull request merged**